.gitignore

```diff
@@ -6,3 +6,4 @@ package-lock.json
 yarn.lock
 vite.config.js.timestamp-*
 .test-output
+new-add-on/
```

Comment:
NIT (i know, pr is wip): we shouldn't add this to the gitignore. You could always use the already provided temp/ folder or generate it outside of this repo